cmd/builder/internal/builder/config.go

```diff
@@ -43,6 +43,8 @@ type Config struct {
 	Replaces     []string     `mapstructure:"replaces"`
 	Excludes     []string     `mapstructure:"excludes"`
 
+	DefaultScheme string `mapstructure:"default_scheme"`
```

Comment:
Scheme is pretty generic, my first thought reading this was "http or https" 😄 

Maybe something like `ConfigDefaultScheme` or `ConfigExpansionDefault` might be specific enough? Though I admit i'm not a fan of either of these options.

Reply:
Best option would be to make Providers a new struct and put it and the Modules in there, but that's another breaking change to deal with.

Reply:
I could go either way on putting it under the `providers` key outside of dealing with the breaking change. On one hand, it does make more sense to group all options in the same section. On the other, it breaks the expectation that `providers` is just another list of modules.

I think `ConfigDefaultScheme` or `DefaultConfigURIScheme` would be okay names, even if they're both long and feel vague. We will ultimately need to rely on documentation to explain what this option does, and ideally not many people need to use it.

Reply:
Moved the option to a new `ResolverSettings` struct in case we ever add more options to that confmap struct that we want to expose via the builder